packages/dds/pact-map/src/interfaces.ts

```diff
@@ -12,7 +12,28 @@ export interface IPactMapEvents extends ISharedObjectEvents {
 	/**
 	 * Notifies when a new value goes pending or has been accepted.
 	 */
-	(event: "pending" | "accepted", listener: (key: string) => void);
+	(event: "pending" | "accepted", listener: (key: string, sequenceNumber: number) => void);
+}
+
+/**
+ * Details of the accepted pact.
+ */
+export interface IAcceptedPactDetails<T> {
```

Comment:
Maybe give the external interface the nice name of `IAcceptedPact` and we can use `IAcceptedPactInternal` for the private interface.